user-guide/key-encryption.rst

```diff
@@ -38,12 +38,12 @@ Let's test if all works:
 .. code-block:: bash
 
     # 1. Encrypt and save the binary content to a file:
-    $ aws kms encrypt --key-id $KMS_KEY_ID --plaintext "<here-you-can-paste-your-pwd>"  --query CiphertextBlob --output text | base64 -d > /tmp/encrypted
+    $ aws kms encrypt --key-id $KMS_KEY_ID --plaintext "<here-you-can-paste-your-pwd>"  --query CiphertextBlob --output text | base64 | tr -d '\n' > /tmp/encrypted
```

Comment:
what about `base64 --decode` does this work on both systems?

Reply:
`base64 --decode` does not work for me.
The proposed solution `base64 | tr -d '\n'` produces proper text output.

`base64 --decode` produces some binary code I do not expect:
```
??%??b,??1??H?
0|0z0uNM`?He.0?Q??U????0?
```

Reply:
actually the ciphertext is supposed to be a cryptic random binary string. 
For me testing your change fails with ´InvalidCiphertextException´